compose/service.py

```diff
@@ -523,6 +530,8 @@ def _get_container_create_options(
             for k in DOCKER_CONFIG_KEYS if k in self.options)
         container_options.update(override_options)
 
+        container_options.update(self.get_logging_options())
+
```

Comment:
You're right, this doesn't look useful since those keys will be removed a few lines later.